sigridci/sigridci/reports/ascii_art_report.py

```diff
@@ -24,36 +24,42 @@ class AsciiArtReport(Report):
     ANSI_YELLOW = "\033[33m"
     ANSI_RED = "\033[91m"
     ANSI_BLUE = "\033[96m"
-    LINE_WIDTH = 70
+    LINE_WIDTH = 80
 
     def __init__(self, output=sys.stdout, ansiColors=True):
         self.output = output
         self.ansiColors = ansiColors
 
     def generate(self, analysisId, feedback, options):
-        publish = options.runMode != RunMode.FEEDBACK_ONLY
```

Comment:
This was just a leftover from when we used to change the exact feedback/columns based on this right? (seems from a quick look back) Good to clean it up then, but lest directly also clean up the import line `from ..publish_options import RunMode` then 🙂

Reply:
Yeah the IDE informed me it wasn't used anymore. I removed the import.

Reply:
Nicely cleaned up 🙂